### ISSUE TITLE
Rebinding the paddles on Switch firmware?

### ISSUE THREAD

**Issue by MatthewCerza** [greggersaurus/SteamControllerConfig#14]
Any chance there is way to do this? I find myself constantly taking screenshots in Rocket League from muscle memory with the paddles on PC.

**Comment by greggersaurus** [comment]
Yes, it should be fairly simple. Look at function updateReports() in https://github.com/greggersaurus/OpenSteamController/blob/master/Firmware/OpenSteamController/src/usb.c (line 1269 as of commit ca3a510f4cb516144bdf3015612a5a9c2adcd889) and remove line "controllerUsbData.statusReport.snapshotButton = getLeftGripState();" This is checking if the Left Paddle is being pressed and using that state to act as the snapshot button. If you remove it and rebuild the Left Paddle will do nothing. 

Please let me know if that isn't  a clear explanation.
